tools/bumbershoot/idpquery/idpquery.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0"?>
+<tool id="idpquery" name="idpQuery" version="0.1.0">
+    <description>Creates text reports from idpDB files.</description>
+    <macros>
+        <xml name="proteinGroupByColumns">
+            <param name="report_columns" type="select" label="Select report columns" display="checkboxes" multiple="true">
+                <option value="Accession">Accession</option>
+                <option value="GeneId">Gene Id</option>
```

Comment:
Can I set a default with selected="true"? I don't think I've tried it yet. The only default would be Accession.